app/api/routes.py

```diff
@@ -173,6 +183,31 @@ def get_attributes(json):
     return (langs, categ)
 
 
+def update_votes(id, vote_direction):
+    resource = None
+    try:
+        resource = Resource.query.get(id)
+
+    except MultipleResultsFound as e:
+        print_tb(e.__traceback__)
+        print(e)
+
+    except NoResultFound as e:
+        print_tb(e.__traceback__)
+        print(e)
+
+    finally:
```

Comment:
I don't think this `finally` statement is necessary. If we're not confident that we're catching all the errors with the `except` blocks, then we should add one that catches `Exception as e` and log the error and return the standard `something went wrong` response.

The code to do this would be:
```python
    except Exception as e:
        print_tb(e.__traceback__)
        print(e)
        return standardize_response(None, None, None, 500)
```

Reply:
change made